.github/workflows/e2e.yml

```
@@ -25,69 +25,124 @@ on:
 env:
   BUILDX_VERSION: latest
   BUILDKIT_IMAGE: moby/buildkit:buildx-stable-1
+  HARBOR_VERSION: v2.7.0
```

```suggestion
  HARBOR_VERSION: 2.7.0
```

Would be nice if we could use the same version format for all the versions specified if possible.